src/fileio.cpp

```diff
@@ -0,0 +1,62 @@
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include "fileio.h"
+
+void import_stl (string filepath) {
+}
+
+void export_stl (World *world, string filepath) {
+  ofstream fs;
+  fs.open(filepath + ".stl");
```

Comment:
d'you have to `touch foo.stl` or similar before running `export_stl`, or does `fs.open` create it for you if it doesn't exist? when does the `!fs.is_open()` case occur?